book/layout.md

```diff
@@ -702,7 +702,7 @@ background has to be drawn *below* that text. Note also that `paint_tree`
 calls `paint` before recursing into the subtree, so the subtree also paints
 on top of this background, as desired.
 
-With the display list filled out, we need to `draw`
+With the display list filled out, we need to draw
```

Comment:
I think we used to call the method itself `draw` but we don't anymore, so this isn't a method reference. The Tk functions we call are all `create_xxx` so `draw` doesn't really make sense.